Content.Server/Resist/ResistLockerSystem.cs

```diff
@@ -0,0 +1,76 @@
+using Content.Shared.Movement;
+using Robust.Shared.GameObjects;
+using Content.Server.Storage.Components;
+using Content.Server.DoAfter;
+using Content.Shared.Popups;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Resist
+{
+    public class ResistLockerSystem : EntitySystem
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<ResistLockerComponent, RelayMovementEntityEvent>(OnRelayMovement);
```

Comment:
Should also subscribe on container changes I think, that way if they ever get out of the container when they're resisting (e.g. it gets unlocked and opened) you can just cancel the do_after.

Reply:
The do_after already cancels on movement so if the locker is opened and the player moves it'll cancel, but I can also subscribe to container change so it cancels if the container becomes unwelded/opened if you like